packages/airnode-utilities/src/evm/gas-prices/gas-prices.ts

```diff
@@ -83,7 +83,7 @@ export const getEip1559GasPricing = async (
   ];
 };
 
-export const getGasPrice = async (options: FetchOptions): Promise<LogsData<GasTarget | null>> => {
+export const getProviderGasPrice = async (options: FetchOptions): Promise<LogsData<GasTarget | null>> => {
```

Comment:
Do we even still need this file? I thought that the oracle will replace it.

Reply:
Maybe more of a question for @aquarat. But I'd say remove it if we're not going to use it right away.

Reply:
Adding them as an alternative strategy is probably a good idea (and wouldn't be much work it seems); none of the new strategies (as superior as they are) replicate the functionality of the above. The new gas pricing strategies are either very simple (constant pricing) or very advanced. I think this as an additional strategy offers a reasonable fall back for a lot of networks at low cost.

We're aiming to serve a lot of weird chains with Airnode (our business _edge_) and I'm scared of us restricting Airnode's functionality and then having do a hack job to support a specific chain. I realise it's a trade-off between too much _dead_ code vs support for weird chains.

I'm not too fussed though :shrug:

Reply:
I mean if we turn it into a strategy and remove the `gas-prices.ts` I'm fine with it. If it's easy to add, it can be even done as part of this PR. @vponline WDYT?

Reply:
(I'm not fussed where the code lives :smile: )

Reply:
Ok sounds good.

> it can be even done as part of this PR. 

Yes, I'll work on adding these next :+1: .